packages/contracts/lib.esm/index.d.ts

```diff
@@ -88,7 +72,7 @@ export declare class BaseContract {
         [name: string]: ContractFunction<BigNumber>;
     };
     readonly populateTransaction: {
-        [name: string]: ContractFunction<PopulatedTransaction>;
+        [name: string]: ContractFunction<TransactionRequest>;
```

Comment:
Is there a reason why we will change the interface to `TransactionRequest`. Can't we use the same `PopulatedTransaction` interface?

Reply:
Pretty sure that we can, just did it by the task's description, where it was written to use `TransactionRequest`.

Reply:
Sorry, it is my fault then. Can we proceed and use `PopulatedTransaction` instead?